app/javascript/packs/commentDropdowns.js

```diff
@@ -69,6 +74,18 @@ const initializeArticlePageDropdowns = () => {
       dropdownTrigger.dataset.initialized = true;
     }
   }
+
+  const dropdownContent = document.getElementById(
+    'comments-sort-dropdown-container',
+  );
+  const dropdownElement = document.getElementById(
+    'toggle-comments-sort-dropdown',
+  );
+
+  if (dropdownContent && !dropdownContent.dataset.initialized) {
+    initializeMemberMenu(dropdownContent, dropdownElement);
+    dropdownContent.dataset.initialized = true;
+  }
```

Comment:
```suggestion
  const sortCommentsDropdownTrigger = document.getElementById(
    'toggle-comments-sort-dropdown',
  );

  if (!sortCommentsDropdownTrigger.dataset.initialized) {
     initializeDropdown({
       triggerElementId: 'toggle-comments-sort-dropdown',
       dropdownContentId: 'comments-sort-dropdown-container',
     });
    sortCommentsDropdownTrigger.dataset.initialized = true
  }
```

This refactors to use the `initializeDropdown` helper used elsewhere. This helper is designed to work with `crayons-dropdown` classes 🙂

Reply:
thanks so much, I made another change that was a bit easier